### DISCUSSION TITLE
eID beim Onlineshoppen und eID für Firmen

### DISCUSSION THREAD

**Discussion by Mannshoch** [e-id-admin/open-source-community#36] [Q&A]
Ich habe Fragen zur eID Nutzung:

Sollte die eID voll eingeführt werden. 

1. Könnte es möglich sein, dass z.B. ein Onlineshop einen user nur durch die eID registriert und von diesem User aber bis auf den hinweis, dass er über 18 ist sonst nichts weiss? 
 Wenn dann ein Käufer Probleme macht, könnte der Onlineshop mit der bei ihnen registrierten eID eine Betreibung eröffnen oder zur Polizei gehen?
 
2. Könnte es gar möglich sein, dass ein Onlineshop auch nicht die Adresse kennt, sondern eine ID auf ein Paket druckt, welche nur durch die Post in eine Adresse aufgelöst werden kann?
 Das hätte theoretisch den Vorteil, dass man die Wohn- oder gegebenenfalls Rechnungsadresse nur an einem Ort anpassen muss und jedes Unternehmen, dass mit der eID arbeitet würde post automatisch an die korrekte Anschrift senden.
 
3. Ist eigentlich geplant, dass eIDs auch für Juristische Personen ausgestellt werden können?

**Comment by christianheimann** [comment]
Danke für das Teilen dieser Gedanken und Fragen:

zu 1. 
Nein, das ist so nicht möglich. Wenn man von einer E-ID-Inhaberin nur abfragt, ob sie über 18 ist, wird man auf Seite der Verifikatorin keine Informationen haben, um wen es sich handelt. Ein Online-Shop wird zumindest Zustellinformationen separat erfragen müssen. Eine Betreibung muss beim zuständigen Betreibungsamt (richtet sich nach dem Wohnsitz der Schuldnerin) eingereicht werden. Dafür ist eine konkrete Adresse erforderlich.
Eine «Anzeige gegen Unbekannt» bei der Polizei macht kaum Sinn. Insbesondere bei geringfügigen Delikten wäre es auch nicht verhältnismässig, hier entsprechende Untersuchungshandlungen zu initiieren.
 
zu 2.
Nein, das ist ebenfalls so nicht möglich. Die E-ID selbst hat keinen direkten Bezug zur Adresse der Inhaberin, die Adresse ist auch nicht Bestandteil der E-ID. Das E-ID-Gesetz erlaubt es nicht, der Post die notwendigen Daten zu übermitteln, die für einen wie von Dir skizzierten Dienst notwendig wären.

zu 3.
Die E-ID ist als Identitätsnachweis von natürlichen Personen konzipiert. Juristische Personen verfügen über keine vergleichbare Identität, sondern handeln durch ihre Organe, welche jeweils natürlich Personen sind. Gerade im elektronischen Geschäftsverkehr – zum Beispiel beim Unterzeichnen von Verträgen – ist die verlässliche Angabe über die Rolle einer Person eine wichtige Information. Lösungsansätze sind daher bereits im Gespräch. Die Umsetzung ist aber nicht Teil des E-ID-Programms.

**Reply by Mannshoch** [reply]
1. Ich denke hier müsste man Lösungen diskutieren. Der wichtigste Grund für ein Unternehmen viele Angaben zu einem Kunden zu haben ist auch die Notwendigkeit gegebenenfalls Juristisch vorgehen zu können. Wenn ein Onlineshop in der Lage wäre nur anhand eines Kryptischen Schlüssels Forderungen geltend machen zu können, dann wäre ein voll anonymes Einkaufen möglich. Gegebenenfalls wäre nicht mal ein Account erforderlich. Besonders interessant wäre es, wenn das Unternehmen die Möglichkeit hätte eine eID zu sperren.

Es könnte sogar möglich sein, gewisse einfache Juristische Verfahren ohne offenlegen von persönlichen Details zu erledigen.
Ab dem Zeitpunkt kann man dann auch wirklich Nutzerdaten von Profil Informationen trennen und wenn ein Unternehmen gerne ein Profil anlegen möchte dann könnte es sich die Erlaubnis beim user auch einhohlen.

3. Alternativ könnte ich mir hier auch vorstellen, dass beim erstmaligen abschliessenden bestellen dem Postdienstleister eine Datenfreigabe für die Adresse gemacht werden muss und dann der Onlineshop nur eine ID des Postdienstleisters auf sein Paket kleben muss.
